assets/js/src/core/components/editor-tabs/editor-tabs.icon-wrapper.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React, { useState } from 'react'
+import { Tooltip } from 'antd'
+
+export interface IconWrapperProps {
+  tabKey: string
+  openTabKey: string | null
+  title: string
+  children: React.ReactNode
+}
+export const IconWrapper = ({ tabKey, openTabKey, title, children }: IconWrapperProps): React.JSX.Element => {
```

Comment:
Can we also add the tooltip, when the Tab has focus via keyboard? :)
Just want make sure that we meet the a11y requirements, if possible.

Reply:
https://github.com/user-attachments/assets/ad318a2e-07aa-45ff-a823-4ab9649b2036

Reply:
As I am not able to access each tab element separately (because of Tabs.TabPane soon being deprecated) 
I have made do with the <Tabs> onFocus/ onBlur combined with some event.target.id text matching. 

When someone is using the Keyboard to tab through the tabs and then switches to navigate via mouse, two tooltips can show at once which I think makes sense (see vid).

Reply:
Yes, I'd agree that it makes sense to show 2 tooltips in that case...